desc/magnetic_fields/_core.py

```diff
@@ -2609,6 +2609,18 @@ def field_line_integrate(
     z0 = z0.flatten()
     x0 = jnp.array([r0, phis[0] * jnp.ones_like(r0), z0]).T
 
+    signBT = np.sign(field.compute_magnetic_field(x0)[0, 1])
+    if signBT < 0:
+        warnings.warn(
+            "Field has negative toroidal component. Field line integration is being "
+            "done in the negative toroidal direction, hence the output is not in phis "
+            "but in negative phis. Use the outputed phis to get the correct "
```

Comment:
Trying to think of a clearer way to phrase this

Field toroidal component is negative, so the field line points in the negative toroidal direction. Field line integration is done along the field line vector direction, hence the output `phi` array of the toroidal angle of the integrated trajectory will be equal to negative of the input `phis` array

I am not sure about this last part, maybe reword as "if you want to integrate along the same field trajectory but in the positive toroidal direction, you can flip the sign of the field", It is not that they are doing something wrong, it is just that the output `phis` will be different (the negative) of the input `phi` array.